pkg/apis/eventing/v1alpha1/rabbitmq_validation.go

```diff
@@ -46,22 +46,19 @@ func (r *RabbitmqBrokerConfig) Validate(ctx context.Context) *apis.FieldError {
 		}
 	}
 
-	var errs *apis.FieldError
 	if r.Spec.RabbitmqClusterReference == nil {
 		return apis.ErrMissingField("rabbitmqClusterReference").ViaField("spec")
 	} else {
 		if r.Spec.RabbitmqClusterReference.Name == "" {
-			errs = errs.Also(apis.ErrMissingField("name").ViaField("rabbitmqClusterReference").ViaField("spec"))
-		}
-		if r.Spec.RabbitmqClusterReference.Namespace == "" {
-			errs = errs.Also(apis.ErrMissingField("namespace").ViaField("rabbitmqClusterReference").ViaField("spec"))
-		}
-		if r.Spec.RabbitmqClusterReference.ConnectionSecret != nil {
+			if r.Spec.RabbitmqClusterReference.ConnectionSecret == nil {
+				return apis.ErrMissingField("name").ViaField("rabbitmqClusterReference").ViaField("spec")
```

Comment:
We can't make this change right now. RabbitmqBrokerConfig does not accept a connection secret in its current state. It'll be a new feature to add that capability.

Reply:
Ok ok. I'll roll it back, i wanted to do a common method to validate both anyway :p! Ty